get_sample_covariates.py

```diff
@@ -124,15 +132,40 @@ def main(
             age_dict_list.append({'sample_id': cpg_id, 'age': age})
     age_df = pd.DataFrame(age_dict_list)
 
+    # genotype PCs
+    pcs_ht_path = dataset_path(f'/large_cohort/{vds_version}/ancestry/scores.ht')
+    pcs_ht = hl.read_table(pcs_ht_path)
```

Comment:
Really strange that this appears to not be returning the full path...  for a sanity check can you throw in a print statement here for `pcs_ht_path`? If the rest of the script is slow and you want to speed it up you can comment out the unnecessary bits. 
```suggestion
    print(pcs_ht)
    pcs_ht = hl.read_table(pcs_ht_path)
```